OracleGoldenGate/21c/bin/deployment-main.sh

```diff
@@ -30,6 +27,14 @@ function abort() {
 :     "${OGG_HOME:?}"
 [[ -d "${OGG_HOME}"            ]] || abort "Deployment runtime, '${OGG_HOME}'. not found."
 
+:     "${ABORT_ON_USER_SCRIPT_ERRORS:=true}"
+:     "${SETUP_USER_SCRIPTS:=${OGG_HOME}/scripts/setup}"
+[[ -d "${SETUP_USER_SCRIPTS}" ]] || mkdir -p "${SETUP_USER_SCRIPTS}"
```

Comment:
These `mkdir` commands can go in the `Dockerfile`

Reply:
This kind of goes back to the question of "Do we allow user defined locations of where the *_USER_SCRIPTS are located".  If we allow user definition, this is a nice safeguard preventing errors in the belly of the script due to missing dirs.

Reply:
Since the DBMS containerized image does not allow a user-defined location for scripts, I would like to use a fixed location here, too.

Reply:
Updated to be the fixed location